scripts/dev/k8s_request_data.py

```diff
@@ -29,6 +29,15 @@ def get_stateful_sets_namespaced(namespace: str) -> dict:
     return sst.to_dict()
 
 
+def get_configmaps_namespaced(namespace: str) -> list:
+    corev1 = client.CoreV1Api()
+    try:
+        config_maps = corev1.list_namespaced_config_map(namespace, pretty="true")
+    except ApiException as e:
+        print("Exception when calling list_namespaced_config_map: %s\n" % e)
+    return config_maps.items
```

Comment:
if the ApiException is raised, config_maps will be None and `.items` will fail with AttributeError

Reply:
Ah you are absolutely right! I'll fix this on all functions in this file!

Reply:
❗ nice catch

Reply:
Should we change the function to return a Optional[dict] and similar and give the caller the burden to check it?
The other way would be to just not catch the exception, but that would interrupting the dumping of logs and mark the task as failed even when it is not. So I'm leaning towards the first.